src/main/java/edu/kis/powp/jobs2d/command/SimpleDriverCommandCountingVisitor.java

```diff
@@ -0,0 +1,26 @@
+package edu.kis.powp.jobs2d.command;
+
+public class SimpleDriverCommandCountingVisitor implements DriverCommandVisitor {
+    private int total;
+
+    @Override
+    public void visit(SetPositionCommand command) {
+    }
+
+    public int getTotal() {
```

Comment:
Put this method at the top or bottom (not between visits). 
I think getCount or getResult would be better